airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -208,11 +221,15 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
         self.arguments = arguments or []
         self.labels = labels or {}
         self.startup_timeout_seconds = startup_timeout_seconds
-        self.env_vars = env_vars or []
+        self.env_vars = convert_env_vars(env_vars) if env_vars else []
+        if pod_runtime_info_envs:
+            self.env_vars.extend([convert_pod_runtime_info_env(p) for p in pod_runtime_info_envs])
         self.env_from = env_from or []
-        self.ports = ports or []
-        self.volume_mounts = volume_mounts or []
-        self.volumes = volumes or []
+        if configmaps:
+            self.env_from.extend([convert_configmap(c) for c in configmaps])
+        self.ports = [convert_port(p) for p in ports] if ports else []
+        self.volume_mounts = [convert_volume_mount(v) for v in volume_mounts] if volume_mounts else []
+        self.volumes = [convert_volume(volume) for volume in volumes] if volumes else []
```

Comment:
These won't issue deprecation warnings, but that's okay because the imports will, right?

Reply:
Right